src/main/scala/org/constellation/ConstellationExecutionContext.scala

```diff
@@ -38,7 +38,17 @@ object ConstellationExecutionContext {
   val callbacks = ExecutionContext.fromExecutor(Executors.newFixedThreadPool(4))
    */
 
-  val bounded: ExecutionContextExecutor = ExecutionContext.fromExecutor(Executors.newWorkStealingPool())
+  val bounded: ExecutionContextExecutor = ExecutionContext.fromExecutor(
+    new ForkJoinPool(
+      {
+        val available = Runtime.getRuntime.availableProcessors
+        if (available == 1) 1 else available - 1
+      },
+      ForkJoinPool.defaultForkJoinWorkerThreadFactory,
+      null,
+      true
+    )
```

Comment:
Do we need to use ForkJoinPool explicitly for bounded thread pool for some reason?
Btw it would be nice to have it named explicitly "bounded", so that we can grep it easily.